### PR TITLE
Stop changefeed automatically if one of processor exits

### PR DESCRIPTION
**Pull request by amyangfei**
### What problem does this PR solve? 

This is part of https://github.com/pingcap/ticdc/issues/664, and tries to release resource if one processor of a changefeed meets error

### What is changed and how it works?

- If a processor exits with error, upload the error message to `task position`.
- If owner finds error in `task position`, stop the changefeed via admin job. 

### Check List 

Tests 

 - Integration test

### Release note

- Stop changefeed automatically if one of processor exits

### CODE REVIEW

File: cdc/model/errors.go [hunk 1]
```diff
@@ -30,3 +30,10 @@ var (
 	ErrCaptureNotExist       = errors.New("capture not exists")
 	ErrUnresolved            = errors.New("unresolved")
 )
+
+// RunningError represents some running error from cdc components, such as processor.
+type RunningError struct {
+	ID        string `json:"id"`
```
Comment (overvenus) [inline cdc/model/errors.go:36]: Could you name it `json:"code"`?

https://github.com/pingcap/tidb/blob/bd28018526397c3ea5e92ee975fea52ec8d6dda7/docs/design/2020-05-08-standardize-error-codes-and-messages.md

Comment (amyangfei) [reply cdc/model/errors.go]: good catch, make the `RunningError` compatible with TiDB standard

**Pull request merged**